package.json

```diff
@@ -176,7 +176,7 @@
         "severity": "error",
         "pattern": [
           {
-            "regexp": "^\\s*(\\[-\\]\\s*.*?)(\\d+)ms\\s*$",
+            "regexp": "^\\s*(?:\\[-\\]\\s)(\\s*.*?)(?:\\s\\d*\\.?\\d*s|\\s\\d*\\.?\\d*ms)\\s*$",
```

Comment:
I wonder if we could simplify this a bit to:

```
"^\\s*(?:\\[-\\]\\s+)(.*?)(?:\\s+\\d+\\.?\\d*\\s*m?s)\\s*$"
```

I don't think we want to include any spaces at the beginning of the message (capture group 1), best to have the first non-capture group scoop up all the spaces after the `[-]`.  Ditto with the second non-capture group - let's have it scoop up all the space at the end of the message.  

Also, shouldn't we expect at least one digit (`\d+`) before the required `s` or `ms`?  Finally, how about we add a `\s*` between the time and the units in case they ever decide that the display looks better with a space in there.  :-)

Reply:
I like it. Great feedback. I'm not great with regular expressions (said everyone on the internet), so your write up was a great learning experience for me. I'll get it updated.